jobs/rocoto/atmensanalrun.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -x
+
+###############################################################
+# Source FV3GFS workflow modules
+. $HOMEgfs/ush/load_fv3gfs_modules.sh
+status=$?
+[[ $status -ne 0 ]] && exit $status
+
+###############################################################
+# set JJOB variables
+export configs="base atmensanal atmensanalrun"
+export EXSCRIPT=${GDASRUNSH:-$HOMEgfs/sorc/gdas.cd/scripts/exgdas_global_atmos_ensanal_run.sh}
```

Comment:
and here.

Reply:
Agreed.  See suggested resolution above.

Reply:
Addressed via [6674720](https://github.com/NOAA-EMC/global-workflow/commit/667472055b8db74e05467f0c7b02eb6a67bce9d1)